specials/SPC18_Matias_SpiesVsMercs.Altis/mission.sqm

```diff
@@ -0,0 +1,7757 @@
+									class Value
+									{
+										class data
+										{
+											singleType="STRING";
+											value="Male11ENG";
+										};
+									};
+								};
+								class Attribute1
+								{
+									property="pitch";
+									expression="_this setpitch _value;";
+									class Value
+									{
+										class data
+										{
+											singleType="SCALAR";
+											value=1.04;
+										};
+									};
+								};
+								nAttributes=2;
+							};
+						};
+						class Item5
+						{
+							dataType="Object";
+							class PositionInfo
+							{
+								position[]={16011.401,18.830313,17277.172};
+								angles[]={0,5.6467333,0};
+							};
+							side="West";
+							flags=5;
+							class Attributes
+							{
+								rank="SERGEANT";
+								description="Spy";
+								isPlayable=1;
+								class Inventory
+								{
+									class uniform
+									{
+										typeName="tacs_Uniform_Garment_LS_BS_BP_BB";
+										isBackpack=0;
+									};
+									map="ItemMap";
+									compass="ItemCompass";
+									watch="ItemWatch";
+									radio="ItemRadio";
+									gps="ItemGPS";
+								};
+							};
+							id=11;
+							type="B_MU_CTRG_TL_F";
+							atlOffset=0.00029182434;
+							class CustomAttributes
+							{
+								class Attribute0
+								{
+									property="speaker";
+									expression="_this setspeaker _value;";
+									class Value
+									{
+										class data
+										{
+											singleType="STRING";
+											value="Male11ENG";
+										};
+									};
+								};
+								class Attribute1
+								{
+									property="pitch";
+									expression="_this setpitch _value;";
+									class Value
+									{
+										class data
+										{
+											singleType="SCALAR";
+											value=1.04;
+										};
+									};
+								};
+								nAttributes=2;
+							};
+						};
+					};
+					class Attributes
+					{
+					};
+					id=5;
+					atlOffset=185.97;
+				};
+				class Item2
+				{
+					dataType="Object";
+					class PositionInfo
+					{
+						position[]={16006.412,20.108738,17275.484};
+						angles[]={0.018951183,0.98687255,0.038069081};
+					};
+					side="Empty";
+					flags=4;
+					class Attributes
+					{
+					};
+					id=20;
+					type="Land_Shoot_House_Panels_Crouch_F";
+				};
+				class Item3
+				{
+					dataType="Object";
+					class PositionInfo
+					{
+						position[]={16012.042,20.04332,17278.896};
+						angles[]={0.023068966,1.0341686,0.03626699};
+					};
+					side="Empty";
+					flags=4;
+					class Attributes
+					{
+					};
+					id=21;
+					type="Land_Shoot_House_Panels_Crouch_F";
+					atlOffset=-0.19860649;
+				};
+				class Item4
+				{
+					dataType="Object";
+					class PositionInfo
+					{
+						position[]={16008.614,20.818506,17283.348};
+						angles[]={6.2808466,6.0567288,0.0024609142};
+					};
+					side="Empty";
+					flags=4;
+					class Attributes
+					{
+						init="spies_extracted = false;";
```

Comment:
Move this to `init.sqf` rather than an init field

Reply:
Done